pjlib/src/pj/os_core_win32.c

```diff
@@ -1093,10 +1093,18 @@ PJ_DEF(pj_status_t) pj_mutex_lock(pj_mutex_t *mutex)
         status = PJ_STATUS_FROM_OS(GetLastError());
 
 #endif
+    if (status == PJ_SUCCESS)
+    {
     LOG_MUTEX((mutex->obj_name, 
-              (status==PJ_SUCCESS ? "Mutex acquired by thread %s" : "FAILED by %s"),
+            "Mutex acquired by thread %s",
               pj_thread_this()->obj_name));
-
+    }
+    else
+    {
+        LOG_MUTEX_WARN((mutex->obj_name,
+           "FAILED by %s",
+            pj_thread_this()->obj_name));
+    }
```

Comment:
As now it is separated, perhaps print the status code as well (e.g: using `PJ_PERROR` instead of `PJ_LOG`)?